content/docs/plugins.md

```diff
@@ -76,7 +76,9 @@ for any technical questions/answers and the announcement your new plugin.
 * [Ada](https://notepad-plus-plus.org/assets/files/NppHelloAdaDemo.zip)
 * [C#](https://github.com/kbilsted/NotepadPlusPlusPluginPack.Net)
 * [D](https://gitlab.com/dokutoku/npp-api)
-* [Delphi](https://sourceforge.net/projects/npp-plugins/files/DelphiPluginTemplate/DelphiPluginTemplate%202.0%20UNICODE/DelphiPluginTemplate2.zip/download)
+* Delphi: 
+  - [Original (ANSI/UNICODE)](https://sourceforge.net/projects/npp-plugins/files/DelphiPluginTemplate/DelphiPluginTemplate%202.0%20UNICODE/DelphiPluginTemplate2.zip/download)
+  - [Modern (32-bit/64-bit)](https://bitbucket.org/rdipardo/delphiplugintemplate/src/default/Readme.textile): this version has the headers necessary to work with Notepad++ v8.3.3
```

Comment:
Suggestion: link here instead:  

    https://bitbucket.org/rdipardo/delphiplugintemplate/get/default.zip

This will fetch a zipball of the current head commit, _sans_ git history.
Self-updating, compact, less work than keeping up with tagged releases &#x2014; and no cartoon labels to find and click through _à la_ GitHub.

Reply:
updated